infra/notifications/src/app.js

```diff
@@ -123,152 +163,86 @@ app.post('/', async (req, res) => {
-        fetch(linkingNotifyEndpoint, {
-          method: 'POST',
-          headers: {
-            Accept: 'application/json',
-            'Content-Type': 'application/json'
-          },
-          body: JSON.stringify({ receivers, token: linkingNotifyToken })
-        })
-      } catch (error) {
-        console.log('Error notifying linking api ', error)
-      }
-    }
+  if (!buyer.id) {
+    console.error(`Error: Missing buyer.id. Skipping ${eventDetailsSummary}`)
+    return
   }
 
-  // Query the DB to get subscriptions from the seller and/or buyer.
-  // Note the filter ensures we do not send notification to the party
-  // who initiated the action:
-  //  - seller initiated action -> only buyer gets notified.
-  //  - buyer initiated action -> only seller gets notified.
-  //  - 3rd party initiated action -> both buyer and seller get notified.
-  const subs = await PushSubscription.findAll({
-    where: {
-      account: {
-        [Sequelize.Op.in]: [buyerAddress, sellerAddress].filter(
-          a => a && a !== party
-        )
-      }
-    }
-  })
+  // ETH address of the party who initiated the action.
+  // Could be the seller, buyer or a 3rd party (ex: arbitrator, affiliate, etc...).
+  if (!returnValues.party) {
+    console.error(`Error: Invalid part, skipping ${eventDetailsSummary}`)
+    return
+  }
 
-  // Filter out redundant endpoints before iterating.
-  await subs
-    .filter((s, i, self) => {
-      return self.map(ms => ms.endpoint).indexOf(s.endpoint) === i
-    })
-    .forEach(async s => {
-      try {
-        const recipient = s.account
-        const recipientRole = recipient === sellerAddress ? 'seller' : 'buyer'
-
-        const message = getNotificationMessage(
-          eventName,
-          party,
-          recipient,
-          recipientRole
-        )
-        if (!message) {
-          return
-        }
-
-        // Send the push notification.
-        // TODO: Add safeguard against sending duplicate messages since the
-        // event-listener only provides at-least-once guarantees and may
-        // call this webhook more than once for the same event.
-        const pushSubscription = {
-          endpoint: s.endpoint,
-          keys: s.keys
-        }
-        const pushPayload = JSON.stringify({
-          title: message.title,
-          body: message.body,
-          account: recipient,
-          offerId: offer.id
-        })
-        await webpush.sendNotification(pushSubscription, pushPayload)
-      } catch (e) {
-        // Subscription is no longer valid - delete it in the DB.
-        if (e.statusCode === 410) {
-          s.destroy()
-        } else {
-          console.log(e)
-        }
-      }
-    })
+  const party = returnValues.party.toLowerCase()
+  const buyerAddress = buyer.id ? buyer.id.toLowerCase() : null
+  const sellerAddress = seller.id ? seller.id.toLowerCase() : null
+
+  console.info(`Info: Processing event ${eventDetailsSummary}`)
+
+  console.log('party:')
```

Comment:
Besides this being logs like @tomlinton points out. It might be more practical to have all 6 lines in 1 line. 
```
logger.info(`party: ${party} buyerAddress: ${buyerAddress} sellerAddress : ${sellerAddress}`)
```
This way it is easier to parse logs in the future in case we will want to extract some kind of statistics

Reply:
Looks like I left in wayyy to many debugging logs. Removed the excess, back to previous logging levels. 
See bec330036d6d5a24eb3aa208b86f982dadd268fd